src/libImaging/Convert.c

```diff
@@ -44,7 +44,8 @@
 #define L(rgb)\
     ((INT32) (rgb)[0]*299 + (INT32) (rgb)[1]*587 + (INT32) (rgb)[2]*114)
```

Comment:
If you've changed ImageColor, shouldn't this be changed as well?
https://github.com/python-pillow/Pillow/blob/fa1deca144210e519acc71e8fa9c8e004fc6f760/src/PIL/ImageColor.py#L137

Reply:
Not the "L" macro itself should be fixed. This macro is used for F mode for example, where no rounding needed. Instead, every place where "L" is used should be analyzed. This is a bit more work which needs more attention and I haven't much time today to do this before the release.

Reply:
Actually, I've fixed ImageColor.getcolor implementation.